docs/ml/vision/mlmodel.md

```diff
@@ -128,55 +140,68 @@ Both the `mlmodel` detector and classifier require that the input and output ten
   - The _input tensor_ must be named `image`
   - The _output tensor_ must be named `probability`
 
-If you [trained your ML model using the Viam app](/ml/train-model/), your `mlmodel` tensors is already named in this fashion, and you can proceed to [test your detector or classifier](#test-your-detector-or-classifier).
+If you [trained your ML model using the Viam app](/ml/train-model/), your `mlmodel` tensors are already named in this fashion, and you can proceed to [test your detector or classifier](#test-your-detector-or-classifier).
 However, if you [uploaded your own ML model](/ml/upload-model/), or are using one from the [Viam registry](https://app.viam.com/registry), you may need to remap your tensor names to meet this requirement, and should follow the instructions to [remap tensor names](#remap-tensor-names).
 
+{{%expand "Click for details on how to remap tensor names." %}}
```

Comment:
Nit: Why hide this content within an expander? I see that it streamlines the page a little, but then loses the flow we get from things like the `[remap tensor names](#remap-tensor-names)` link that can't link within the expander. Personally, I'd present this section as a regular old subheader `### Remap tensor names` without using the expander, but up to you if you want to retain.

Reply:
seems small enough to take out of the expander